src/hip_memory.cpp

```diff
@@ -1166,6 +1166,11 @@ hipError_t hipMemcpyHtoD(hipDeviceptr_t dst, void* src, size_t sizeBytes) {
 
     hipError_t e = hipSuccess;
 
+    if(dst==NULL || src==NULL)
+	{
+	e=hipErrorInvalidValue;
+	return ihipLogStatus(e);
+	}
```

Comment:
Here and below: 1. formatting: left parenthesis should be placed on the same line as 'if' statement; 2. Excessive assignment.

Reply:
I didn't get the 2nd point. can u please let me know.